go.mod

```diff
@@ -0,0 +1,79 @@
+module github.com/GoogleContainerTools/skaffold
```

Comment:
I ran `GO111MODULE=on go mod tidy` and `GO111MODULE=on go mod vendor` and it removed a lot of dependencies.